includes/Modules/AdSense/AMP_Tag.php

```diff
@@ -135,10 +139,15 @@ private function amp_content_add_auto_ads( $content ) {
 
 		$this->adsense_tag_printed = true;
 
+		$snippet_comment_begin = sprintf( "\n<!-- %s -->\n", esc_html__( 'Google AdSense AMP snippet added by Site Kit', 'google-site-kit' ) );
+		$snippet_comment_end   = sprintf( "\n<!-- %s -->\n", esc_html__( 'End Google AdSense AMP snippet added by Site Kit', 'google-site-kit' ) );
+
 		return sprintf(
-			'<amp-auto-ads type="adsense" data-ad-client="%s"%s></amp-auto-ads> %s',
+			'%s<amp-auto-ads type="adsense" data-ad-client="%s"%s></amp-auto-ads>%s %s',
+			$snippet_comment_begin,
 			esc_attr( $this->tag_id ),
 			$this->get_tag_blocked_on_consent_attribute(),
+			$snippet_comment_end,
```

Comment:
This works but formatted strings get a bit hard to read the more positional placeholders you add to them. Rather than changing this, how about we store the formatted tag string in a variable and then apply the wrapping comments on that? It adds an extra step but makes it easier to read or change later.

Reply:
@aaemnnosttv But wouldn't the $content come after the End Snippet comment? Bit confused about that. 

Which one is correct? 

```php
                $snippet_comment_begin = sprintf( "\n<!-- %s -->\n", esc_html__( 'Google AdSense AMP snippet added by Site Kit', 'google-site-kit' ) );
		$snippet_comment_end   = sprintf( "\n<!-- %s -->\n", esc_html__( 'End Google AdSense AMP snippet added by Site Kit', 'google-site-kit' ) );

		$tag =  sprintf(
			'<amp-auto-ads type="adsense" data-ad-client="%s"%s></amp-auto-ads>%s',
			esc_attr( $this->tag_id ),
			$this->get_tag_blocked_on_consent_attribute(),
			$content
		);

		return $snippet_comment_begin . $tag . $snippet_comment_end;
```

```php
                $snippet_comment_begin = sprintf( "\n<!-- %s -->\n", esc_html__( 'Google AdSense AMP snippet added by Site Kit', 'google-site-kit' ) );
		$snippet_comment_end   = sprintf( "\n<!-- %s -->\n", esc_html__( 'End Google AdSense AMP snippet added by Site Kit', 'google-site-kit' ) );

		$tag =  sprintf(
			'<amp-auto-ads type="adsense" data-ad-client="%s"%s></amp-auto-ads>',
			esc_attr( $this->tag_id ),
			$this->get_tag_blocked_on_consent_attribute(),
		);

		return $snippet_comment_begin . $tag . $snippet_comment_end . $content;
```

Reply:
Yes, the second one (`$snippet_comment_end . $content`) please 👍

Reply:
Done and merge conflict fixed. fill free to have another look :)